tests/generative/test_query_model.py

```diff
@@ -63,7 +63,7 @@
     value_strategies,
 )
 settings = dict(
-    max_examples=(int(os.environ.get("GENTEST_EXAMPLES", 100))),
+    max_examples=(int(os.environ.get("GENTEST_EXAMPLES", 200))),
```

Comment:
Do I gather that you found it was sometimes hitting as many as two errors in 100 examples, but you haven't ever seen three or more over 200? This seems like it's right on the border, so it's bound to be a bit flaky isn't it? Maybe 1% is too tight a constraint.

Reply:
Locally it was doing OK with 200, but it's failed again on that check.  I didn't want to make the overall test runs slower by upping the examples too much; maybe it's better to have make the constraint dependent on the number of examples, so it's e.g. 5% if GENTEST_EXAMPLES is < 500

Reply:
Or turn off that check for the build and only have it on for the long-running overnight tests.